walkthroughs/howto-k8s-appmesh-load-test/README.md

```diff
@@ -0,0 +1,66 @@
+# AppMesh K8s Load Test
+This walkthrough demonstrates how to load test AppMesh on EKS. It can be used as a tool for further load testing in different mesh configuration. We use [Fortio](https://github.com/fortio/fortio) to generate the load. Currently, this walkthrough only focuses AppMesh on EKS. 
+
+## Step 1: Prerequisites
+1. [Walkthrough: App Mesh with EKS](../eks/)
+   1. Make sure you have "appmesh-prometheus" installed. You may follow this [live docs](https://aws.github.io/aws-app-mesh-controller-for-k8s/) _Guide_ section for further installation support.
+   2. Alternatively, you can follow this doc: [Getting started with AWS App Mesh and Kubernetes](https://docs.aws.amazon.com/app-mesh/latest/userguide/getting-started-kubernetes.html) to install appmesh-controller and EKS cluster using `eksctl`.
+2. Clone this repository and navigate to the `walkthroughs/howto-k8s-appmesh-load-test` folder, all the commands henceforth are assumed to be run from the same directory as this README.
+3. Make sure you have the latest version of [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed.
+4. This test requires Python 3 or later (tested with Python 3.9.6). So make sure you have [Python 3](https://www.python.org/downloads/) installed.  
+5. Load test results will be stored into S3 bucket. So, in `scripts/constants.py` give your `S3_BUCKET` a unique name. 
+6. In case you get `AccessDeniedException` (or any kind of accessing AWS resource denied exception) while creating any AppMesh resources (e.g., VirtualNode), don't forget to authenticate with your AWS account.
+
+
+## Step 2: Set Environment Variables
+We need to set a few environment variables before starting the load tests.
+
+```bash
+export CONTROLLER_PATH=<Path to the controller directory e.g., /home/userName/workplace/appmesh-controller/aws-app-mesh-controller-for-k8s>
+export CLUSTER_NAME=<Name of the EKS cluster, e.g., appmeshtest>
+export KUBECONFIG=<If eksctl is used to create the cluster, the KUBECONFIG will look like: ~/.kube/eksctl/clusters/cluster-name>
+export AWS_REGION=us-west-2
+export VPC_ID=<VPC ID of the cluster, can be found using:  aws eks describe-cluster --name $CLUSTER_NAME | grep 'vpcId'>
+```
+
+
+
+## Step 3: Configuring the Load Test
+All parameters of the mesh, load tests, metrics can be specified in `config.json`
+
+`backends_map` -: The mapping from each Virtual Node to its backend Virtual Services. For each unique node name in `backends_map`, 
+a VirtualNode, Deployment, Service and VirtualService (with its VirtualNode as its target) are created at runtime.
+
+`load_tests` -: Array of different test configurations that need to be run on the mesh. `url` is the service endpoint that Fortio (load generator) should hit.
+
+`metrics` -: Map of metric_name to the corresponding metric PromQL logic
+
+## Step 4: Running the Load Test
+Run the driver script using the below command -:
+> sh scripts/driver.sh
```

Comment:
I think this is going to require setup of the controller dev environment, tools such as Ginkgo. We probably need some instructions for that.